openassessment/templates/openassessmentblock/oa_base.html

```diff
@@ -4,7 +4,7 @@
     <div class="openassessment problem">
         <div class="wrapper--grid">
             {% if title %}
-                <h2 class="openassessment__title problem__header">{% trans title %}</h2>
+                <h3 class="openassessment__title problem__header">{% trans title %}</h3>
```

Comment:
This is the title element that is set in studio and per Mark's request it should be an H3 pending the changes to the Unit title to h2.

I will look at what is there once this is replaced.